Core/Prognetics.Saga.Parsers.Core/Model/ReaderConfiguration.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Prognetics.Saga.Parsers.Core.Model
+{
+    public class ReaderConfiguration
+    {
+        public string ParserType { get; set; }
+
+        public string Uri { get; set; }
```

Comment:
The URI refers to an internet standard. Using it as a file path can be confusing